### DISCUSSION TITLE
Getter selector for a nested attribute?

### DISCUSSION THREAD

**Discussion by simon3495** [udecode/zustand-x#88] [Q&A]
```ts
interface Data {
    nestedAttribute: {
        a: string;
        b: string;
    }
}
const store = createStore("Data")<Data>...;
```

What's the correct way to only subscribe to changes in `nestedAttribute.a`? 

What's the difference between these two?

```ts
const x = repoStore.useTrackedStore().nestedAttribute.a; 
const y = repoStore.useStore.nestedAttribute.a; 
```

The docs are not clear on the difference: https://zustand-x.udecode.dev/#md:global-tracked-hook-selectors

**Comment by zbeyens** [comment]
See https://github.com/dai-shi/react-tracked

**Reply by simon3495** [reply]
After reading about react-tracked, I still don't know the difference between the two:
```ts
const x = repoStore.useTrackedStore().nestedAttribute.a; 
const y = repoStore.useStore.nestedAttribute.a; 
```
Similarities:
* They both only rerender if `nestedAttribute.a` changes
* ???

Differences: 
* ???

**Reply by zbeyens** [reply]
I guess the difference is, with `repoStore.useStore.nestedAttribute.a`, the component re-renders if `nestedAttribute.b` is updated.
